House-of-glass/src/Contexts/Product/ProductCompose.jsx

```diff
@@ -0,0 +1,59 @@
+const sortby = ({ sortBy }, array) => {
+  switch (sortBy) {
+    case "HIGHTOLOW":
+      return sortBy === "HIGHTOLOW"
+        ? [...array].sort((a, b) => Number(b.price) - Number(a.price))
+        : [...array];
+    case "LOWTOHIGH":
+      return sortBy === "LOWTOHIGH"
+        ? [...array].sort((a, b) => Number(a.price) - Number(b.price))
+        : [...array];
+    default:
+      return [...array];
+  }
+};
+
+const fastdelivery = ({ fastDelivery }, array) => {
+  let filteredList = fastDelivery
+    ? array.filter((item) => item.fastDelivery)
+    : array;
```

Comment:
Could have directly returned as filter does not mutate the input array

Reply:
Yes, makes sense. Not sure why I did this here. Have corrected it in latest commit. Thank You.